Pollo/Models/ListDiffableModels/Poll.swift

```diff
@@ -13,52 +13,24 @@ enum PollState: String, Codable {
     case live, ended, shared
 }
 
-enum QuestionType: String, CustomStringConvertible, Codable {
-
-    case multipleChoice
-
-    var description: String {
-        switch self {
-        case .multipleChoice: return StringConstants.multipleChoice
-        }
-    }
-
-    var descriptionForServer: String {
-        switch self {
-        case .multipleChoice: return Identifiers.multipleChoiceIdentifier
-        }
-    }
-}
-
 class PollResult: Codable, Equatable {
 
     static func == (lhs: PollResult, rhs: PollResult) -> Bool {
         return lhs.letter == rhs.letter && lhs.text == rhs.text && lhs.count == rhs.count
     }
 
     var count: Int?
-    var letter: String?
+    var letter: Int
```

Comment:
probably would be a good idea to update this variable name so we don't intuitively deduce letter's type as a string

Reply:
I have to keep it labeled as `letter` to match the API models on the backend side, otherwise i'd have to rename the keyvalue for Codable conformance

Reply:
Fixed! backend renamed the field :)